### PR TITLE
Update Template Variables page and Notebooks page

### PR DESCRIPTION
**Pull request by estherk15**
### What does this PR do?
  - Updated the Saved Views sections with video walkthrough
  - Removed unused image files from repo
  - Added image for dynamic template variables w/ updated info
  - Updated the Widgets section w/ new image
  - Add redirect in Notebooks page for Template variables
  
### Motivation
[DOCS-4145](https://datadoghq.atlassian.net/browse/DOCS-4145)

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/dashboards/template_variables.md [hunk 1]
```diff
@@ -58,45 +58,41 @@ On log, APM, and RUM widgets, you can use wildcards in the middle of a value (fo
...
 To delete a view, click on the saved views dropdown and choose *Manage views...*. From there, a popup with your saved views is displayed with a trash bin icon next to each view. Click the appropriate trash bin icon to delete a view.
 
-{{< img src="dashboards/template_variables/manage_views.png" alt="Manage View Popup" style="width:75%;">}}
-
 #### Modify
 
-To modify the *(Default Value)* view, click on the pencil icon and update the template variable values. Then click *Done* to save. If any values in the other views are changed, save the values as a new view, and then delete the original view.
+To modify the *Default view*, click on the pencil icon and update the template variable values. Then click *Done* to save. If any values in the other views are changed, save the values as a new view, and then delete the original view.
 
 ## Use
```
Comment (alai97) [inline content/en/dashboards/template_variables.md:77]:
```suggestion
## Usage
```

File: content/en/dashboards/template_variables.md [hunk 2]
```diff
@@ -58,45 +58,41 @@ On log, APM, and RUM widgets, you can use wildcards in the middle of a value (fo
...
 
 Template variables are used in widgets and event overlays.
 
 ### Widgets
 
-When creating or editing a widget, existing template variables display as options in the `from` field. For example, if you create the template variable `env`, the option `$env` is available.
-
-After the widget is saved, the value of the template variable is the one selected from the top of your dashboard:
+When creating or editing a widget, existing template variables display as options in the `from` field. For example, if you configure the template variable `environment`, the option `$environment` is available as a dynamic variable in the widget.
```
Comment (alai97) [inline content/en/dashboards/template_variables.md:83]:
```suggestion
When creating or editing a widget, existing template variables display as options in the `from` field. For example, if you configure the `environment` template variable, the `$environment` option is available as a dynamic variable in the widget. 
```

File: content/en/dashboards/template_variables.md [hunk 3]
```diff
@@ -58,45 +58,41 @@ On log, APM, and RUM widgets, you can use wildcards in the middle of a value (fo
...
-
-After the widget is saved, the value of the template variable is the one selected from the top of your dashboard:
+When creating or editing a widget, existing template variables display as options in the `from` field. For example, if you configure the template variable `environment`, the option `$environment` is available as a dynamic variable in the widget. 
 
-{{< img src="dashboards/template_variables/selecting_template_variables.png" alt="Selecting template variables" style="width:75%;">}}
+{{< img src="dashboards/template_variables/dynamic_template_variable.png" alt="Template variable can be set dynamically in widgets" style="width:100%;">}}
 
-To query based on just a prefix or suffix, use a wildcard character (`*`) at the beginning or end of the template variable value. For example, find all regions that begin with `us` using `us*`, or all emails under the `example.com` domain with `*@example.com`.
+Selecting the `environment` value of "production", dynamically scopes widgets with the `$environment` variable to the production environment.
```
Comment (alai97) [inline content/en/dashboards/template_variables.md:87]:
```suggestion
Selecting **production** for the `environment` value dynamically scopes widgets with the `$environment` variable to the production environment.
```

File: content/en/dashboards/template_variables.md [hunk 4]
```diff
@@ -58,45 +58,41 @@ On log, APM, and RUM widgets, you can use wildcards in the middle of a value (fo
...
 
 To include just the value in the query, append it with the syntax `$<TEMPLATE_VARIABLE_NAME>.value`. For example, with a template variable named `service`, use `env:staging-$service.value`.
 
 #### Associated template variables
-When selecting a template variable value, the **Associated Values** and **Other Values** sections are displayed. Associated values are calculated from the other template variable values selected on the page, and seamlessly identify the related values without any configuration.
 
-{{< img src="dashboards/template_variables/associated_template_variables.png" alt="Associated template variables" style="width:75%;">}}
+When selecting a template variable value, associated values are displayed at the top of the selector. Associated values are calculated from the other template variable values selected on the page, and seamlessly identify the related values without any configuration.
```
Comment (alai97) [inline content/en/dashboards/template_variables.md:95]:
```suggestion
When selecting a template variable value, associated values are displayed at the top of the selector. Associated values are calculated from other template variable values selected on the page, and seamlessly identify the related values without any configuration.
```

File: content/en/notebooks/_index.md [hunk 5]
```diff
@@ -153,7 +153,11 @@ Notebooks can be set to automatically snapshot graphs that might expire. Enable
 
 To share a version of your notebook with snapshots, click **View graph snapshots** in the notebook header and copy the URL, or append `&view=snapshots` to the URL of any notebook that has snapshots enabled.
 
-## Sharing Notebooks
+### Template variables
+
+Notebooks support template variables, dynamically scope visualizations by adding and selecting template variable values. For more information, see [Template Variables][11].
```
Comment (alai97) [inline content/en/notebooks/_index.md:158]:
```suggestion
Notebooks support template variables. Dynamically scope visualizations by adding and selecting template variable values. For more information, see [Template Variables][11].
```

File: content/en/notebooks/_index.md [hunk 6]
```diff
@@ -153,7 +153,11 @@ Notebooks can be set to automatically snapshot graphs that might expire. Enable
...
+
+## Sharing notebooks
```
Comment (alai97) [inline content/en/notebooks/_index.md:160]:
```suggestion
## Share notebooks
```

File: content/en/dashboards/template_variables.md [hunk 7]
```diff
@@ -58,45 +58,41 @@ On log, APM, and RUM widgets, you can use wildcards in the middle of a value (fo
...
+To the left of the template variables on a dashboard, there is a dropdown listed as *Saved Views*. When you make a change to a template variable value, the value is not automatically saved to a view.
 
-To the left of the template variables on a dashboard, there is a dropdown listed as *(Default Value)*. When you make a change to a template variable value, the value is not automatically saved to a view.
-To save the current values of the template variables in a view, click on the dropdown menu and click *Save selections as view*. From there, you are prompted to enter a unique name for the view. After saving, this view is listed in the dropdown menu. Click on this view to retrieve the previously saved values for the template variables.
+To save the current values of the template variables in a view, click on the *Saved Views* dropdown menu and click *Save selections as view*. From there, you are prompted to enter a unique name for the view. After saving, this view is listed in the dropdown menu. Click on this view to retrieve the previously saved values for the template variables.
```
Comment (alai97) [inline content/en/dashboards/template_variables.md:67]:
```suggestion
To save your current template variables' values in a view, select **Save selections as view** from the **Saved Views** dropdown menu. Enter a unique name for the view and click **Save**. 

Your saved view appears in the dropdown menu. Click on the view to retrieve your previously saved template variable values.
```

File: content/en/dashboards/template_variables.md [hunk 8]
```diff
@@ -58,45 +58,41 @@ On log, APM, and RUM widgets, you can use wildcards in the middle of a value (fo
...
 #### Delete
 
 To delete a view, click on the saved views dropdown and choose *Manage views...*. From there, a popup with your saved views is displayed with a trash bin icon next to each view. Click the appropriate trash bin icon to delete a view.
 
-{{< img src="dashboards/template_variables/manage_views.png" alt="Manage View Popup" style="width:75%;">}}
-
 #### Modify
 
-To modify the *(Default Value)* view, click on the pencil icon and update the template variable values. Then click *Done* to save. If any values in the other views are changed, save the values as a new view, and then delete the original view.
+To modify the *Default view*, click on the pencil icon and update the template variable values. Then click *Done* to save. If any values in the other views are changed, save the values as a new view, and then delete the original view.
```
Comment (alai97) [inline content/en/dashboards/template_variables.md:75]:
```suggestion
To modify the **Default view**, click on the pencil icon and update the template variable values. Then click **Done** to save. If any values in the other views are changed, save the values as a new view, and then delete the original view.
```

File: content/en/dashboards/template_variables.md [hunk 9]
```diff
@@ -58,45 +58,41 @@ On log, APM, and RUM widgets, you can use wildcards in the middle of a value (fo
 
 ### Saved views
 
+{{< img src="dashboards/template_variables/saved_views.mp4" alt="Walk through of how to create, delete, and modify saved views" style="width:100%;" video="true">}}
+
 #### Create
 
-{{< img src="dashboards/template_variables/default_view.png" alt="Default Saved View" style="width:85%;">}}
+To the left of the template variables on a dashboard, there is a dropdown listed as *Saved Views*. When you make a change to a template variable value, the value is not automatically saved to a view.
```
Comment (alai97) [inline content/en/dashboards/template_variables.md:65]:
```suggestion
Click on the **Saved Views** dropdown menu to the left of the template variables in your dashboard. When you update a template variable value, the value does not automatically save to a view.
```

File: content/en/dashboards/template_variables.md [hunk 10]
```diff
@@ -58,45 +58,41 @@ On log, APM, and RUM widgets, you can use wildcards in the middle of a value (fo
...
 
-To the left of the template variables on a dashboard, there is a dropdown listed as *(Default Value)*. When you make a change to a template variable value, the value is not automatically saved to a view.
-To save the current values of the template variables in a view, click on the dropdown menu and click *Save selections as view*. From there, you are prompted to enter a unique name for the view. After saving, this view is listed in the dropdown menu. Click on this view to retrieve the previously saved values for the template variables.
+To save the current values of the template variables in a view, click on the *Saved Views* dropdown menu and click *Save selections as view*. From there, you are prompted to enter a unique name for the view. After saving, this view is listed in the dropdown menu. Click on this view to retrieve the previously saved values for the template variables.
 
 #### Delete
 
 To delete a view, click on the saved views dropdown and choose *Manage views...*. From there, a popup with your saved views is displayed with a trash bin icon next to each view. Click the appropriate trash bin icon to delete a view.
```
Comment (alai97) [inline content/en/dashboards/template_variables.md:71]:
```suggestion
To delete a view, click on the saved views dropdown and choose **Manage views...**. From there, a popup with your saved views is displayed with a trash bin icon next to each view. Click the appropriate trash bin icon to delete a view.
```

**Review by alai97 — CHANGES_REQUESTED** [review]

File: content/en/dashboards/template_variables.md (hunk 3 above)
Comment (estherk15) [reply content/en/dashboards/template_variables.md:87]: I wasn't sure how to let the customer know that "production" is an example value. Would an italic work better here?

**Review by alai97 — APPROVED** [review]

**Pull request merged**